mobile/geth.go

```diff
@@ -38,6 +38,13 @@ import (
 	whisper "github.com/ethereum/go-ethereum/whisper/whisperv6"
 )
 
+// I am intentionally duplicating these constants different from downloader.SyncMode integer values, to ensure the
+// backwards compatibility where the mobile node defaults to LightSync.
+const SyncModeUnset = 0 // will be treated as SyncModeLightSync
```

Comment:
In golang, it's common to use `iota` for these things: https://github.com/golang/go/wiki/Iota

Reply:
Thanks. Fixed.

Reply:
Thanks. Fixed.